rustfmt.toml

```diff
@@ -0,0 +1,32 @@
+# Run using `rustfmt --write-mode overwrite *.rs` in the
+# root of the src directory. You may still get some
+# formatting errors (whitespace etc) which should be
+# fixed manually before committing.
+
+# Max line length that will be parsed by rustfmt
+max_width = 160
```

Comment:
Can we re-run with this set to 120? I think 160 is a little much

Reply:
Not sure if the comment is confusing here...160 is the max line length that will be parsed by the `rustfmt` tool _not_ the max line length allowed in the code base. (I hope that makes sense!)

So, under current settings: any line up to 160 characters will be parsed and formatted, anything longer will be an error. 160 is overkill now but rustfmt was unable to parse some of the codebase without going this high. I'll bring it down to 120 and then from now on, when you run rustfmt it will complain at you if you go over 120 characters (which you will need to manually fix each time).

Reply:
looks like its mainly test code that `#[cfg_attr(rustfmt, rustfmt_skip)]` doesn't seem to be working for...

Reply:
ah I think I misunderstood the purpose of max_width. It should be fine to set to 160 in that case. Sorry about the mix up